language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules_daml:daml.bzl", "daml_compile")
+load("@os_info//:os_info.bzl", "is_windows")
+
+daml_compile(
+    name = "daml2ts-test",
+    main_src = "daml/Main.daml",
+    srcs = glob(["daml/**/*.daml"]),
+)
+
+# This is needed to be able to call `$(location ...)` in rules.
+# Otherwise I have not found a way to express the conditional within the
+# `$(location ...)` substitution.
+alias(
+    name = "yarn",
+    actual = "@nodejs//:bin/yarn.cmd" if is_windows else "@nodejs//:bin/yarn",
+)
+
+sh_test(
```

Comment:
I expect/hope that this will turn into a proper tasty test suite at some point since I’m scared of bash but fine to leave it for now.

Reply:
Yes, it will. Right now, bash was the fastest way to get this into master before the PR gets too big.